Dockerfile.build.tpl

```diff
@@ -0,0 +1,38 @@
+# Dockerfile.build represents the build environment of the driver, used during
```

Comment:
Please, remove all comments from the template and leave only any comment you need that is specific to your Dockerfile, or remove comments altogether.